### ISSUE TITLE
503 Service Unavailable on homepage

### ISSUE THREAD

**Issue by DenisFlorin** [10up/ElasticPress#2198]
**Describe the bug**
I am getting this error when visiting the homepage of my website, while I'm logged in and visit the query monitor tab.

**Steps to Reproduce**

1. Visit homepage'
2. Click on query monitor tab
3. When inspecting with the WP plugin "Query monitor" i get this error
4. See error in tab HTTP API errors

Expected behavior
I expect to see no errors.

Screenshots
![image](https://user-images.githubusercontent.com/38504865/117180970-6f342c80-addd-11eb-9cdc-0146797fe4be.png)

Error seen in debug bar elastic press
`{
    "error": {
        "root_cause": [],
        "type": "search_phase_execution_exception",
        "reason": "",
        "phase": "fetch",
        "grouped": true,
        "failed_shards": [],
        "caused_by": {
            "type": "too_many_buckets_exception",
            "reason": "Trying to create too many buckets. Must be less than or equal to: [10000] but was [10001]. This limit can be set by changing the [search.max_buckets] cluster level setting.",
            "max_buckets": 10000
        }
    },
    "status": 503
}`

Query body
`{
    "from": 0,
    "size": 11,
    "sort": [
        {
            "post_date": {
                "order": "desc"
            }
        }
    ],
    "query": {
        "match_all": {
            "boost": 1
        }
    },
    "post_filter": {
        "bool": {
            "must": [
                {
                    "terms": {
                        "post_type.raw": [
                            "post"
                        ]
                    }
                },
                {
                    "terms": {
                        "post_status": [
                            "publish",
                            "acf-disabled"
                        ]
                    }
                }
            ]
        }
    },
    "aggs": {
        "terms": {
            "filter": {
                "bool": {
                    "must": [
                        {
                            "terms": {
                                "post_type.raw": [
                                    "post"
                                ]
                            }
                        },
                        {
                            "terms": {
                                "post_status": [
                                    "publish",
                                    "acf-disabled"
                                ]
                            }
                        }
                    ]
                }
            },
            "aggs": {
                "category": {
                    "terms": {
                        "size": 10000,
                        "field": "terms.category.slug"
                    }
                },
                "post_tag": {
                    "terms": {
                        "size": 10000,
                        "field": "terms.post_tag.slug"
                    }
                },
                "post_format": {
                    "terms": {
                        "size": 10000,
                        "field": "terms.post_format.slug"
                    }
                },
                "ep_custom_result": {
                    "terms": {
                        "size": 10000,
                        "field": "terms.ep_custom_result.slug"
                    }
                }
            }
        }
    }
}`

**Comment by felipeelia** [comment]
Hi @DenisFlorin,

That is related to the size of the aggregation buckets. Can you try to add the following code to a custom plugin or to your theme's functions.php and reload the page?

add_filter(   'ep_facet_taxonomies_size', function() { return 1000; } );

You can also try to increase that limit as explained [here](https://github.com/grafana/grafana/issues/17327).

**Comment by DenisFlorin** [comment]
Hi @felipeelia,

I added that filter, and the error is not present anymore. If I keep it like that, will it impact the search or the website?

Also, if I increase the limit,  will it affect the performance of the website?

Thank you,

**Comment by jeffpaul** [comment]
@DenisFlorin we recently added a GitHub Action to help us maintain issues in this repo wherein it will auto-close issues that aren't responded to within a certain period of time once a `reporter feedback` label is applied.  It appears to have mistakenly closed your issue, but don't worry we're keeping an eye on your issue and will reopen it once we've been able to triage why the GitHub action mistakenly closed your issue.  Apologies for the confusion and I appreciate your patience... thanks!
